.automation.conf/tempest/load-lists/tempest-full

```diff
@@ -0,0 +1,1601 @@
+tempest.api.object_storage.test_object_slo.ObjectSloTest.test_upload_manifest[id-2c3f24a6-36e8-4711-9aa2-800ee1fc7b5b]
```

Comment:
I do wonder how portable this list is between different environments. Probably something to ask about in Slack. i.e. do different environments tend to have different "full" test lists, and if so, should they?

Reply:
I've asked about this in slack